src/Sarif/ExtensionMethods.cs

```diff
@@ -127,6 +127,15 @@ public static Message ToMessage(this string text)
             return new Message { Text = text };
         }
 
+        public static string ToCamelCase(this string name)
```

Comment:
>ToCamelCase [](start = 29, length = 11)

I really don't see this as valuable API in the low-level SDK surface area. We should always be hesitant to create a general API at a low level with a single caller. Just as we look askance at a base type with only a single derived instance. When you add an API at a low-level, this implies support, docs, etc. Also, raises the question: if you're in the business of camel-casing, where's my pascal-casing utility. So my suggestion is to co-locate this helper with the calling code (and wait for another example of its general utility before putting it here).